ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -163,11 +395,16 @@ class ServicesHomeScreen extends React.Component<Props, State> {
   }
 
   private showChooserAreasOfInterestModal = () => {
-    const { allOrganizations, hideModal, organizationsSelected } = this.props;
+    const {
+      selectableOrganizations,
+      hideModal,
+      selectedOrganizations,
+      isLoading
+    } = this.props;
     this.props.showModal(
       <ChooserListContainer<Organization>
-        items={allOrganizations}
-        initialSelectedItemIds={some(new Set(organizationsSelected))}
+        items={selectableOrganizations}
+        initialSelectedItemIds={some(new Set(selectedOrganizations))}
```

Comment:
```suggestion
        initialSelectedItemIds={some(new Set(selectedOrganizations || []))}
```
just to keep it more formal (since **selectedOrganizations** could be undefined)

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1269/commits/cb05c6b87daa3bd2ed57b135557e530f77d130af